syntax = "proto3";
package smartbox;

service LightController {
	rpc set_light (LightRequest) returns (LightResponse) {}
}

message LightRequest {
	enum LightSetting{
	STATUS = 0;
	ON = 1;
	OFF = 2;
	}
	LightSetting light = 1;
}

message LightResponse {
	enum LightStatus {
		ON = 0;
		OFF = 1;
	}
	LightStatus status = 1;
}